File: HZW.ZHCG2/HZW.ZHCG.WebAPI/app/model/Categors.js
Ext.define('TianZun.model.Categors', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'ID', type: 'int' },
        { name: 'Name', type: 'string' },
        {
            name: 'createdTime', type: 'string', convert: function (value) {
                var enddate = Ext.Date.format(new Date(value), "Y-m-d");
                return enddate;
            }
        },
        { name: 'isonline', type: 'int'},
        {
            name: 'isonlinestring', type: 'string', convert: function (value,record) {
                if (record.get('isonline') == 1) {
                    return '上线';
                }
                else {
                    return '下线';
                }
            }
        },
        { name: 'BigID', type: 'int' },
        { name: 'BigName', type: 'string' },
        { name: 'Createuserid', type: 'int' },
        { name: 'SeqNo', type: 'int' }
    ]
});

File: HZW.ZHCG2/HZW.ZHCG.Model/Categors.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HZW.ZHCG.Model
{
    public class Categors
    {
        /// <summary>
        /// 编号
        /// </summary>
        public int ID { get; set; }

        /// <summary>
        /// 名称
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// 排序
        /// </summary>
        public int? SeqNo { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        public Nullable<System.DateTime> createdTime { get; set; }

        /// <summary>
        /// 是否上线
        /// </summary>
        public int? isonline { get; set; }

        /// <summary>
        /// 大类名称
        /// </summary>
        public string BigName { get; set; }

        /// <summary>
        /// 大类编号
        /// </summary>
        public int? BigID { get; set; }

        /// <summary>
        /// 创建人
        /// </summary>
        public int Createuserid { get; set; }
    }
}
